cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/SystemArguments.java

```diff
@@ -87,6 +109,21 @@ private static Integer getPositiveInt(Map<String, String> map, String key, Strin
     return null;
   }
 
+  private static void setLogLevels(Map<String, String> args, LogAppenderInitializer initializer) {
```

Comment:
I don't like method like `setLogLevel` and `setLogLevels`. It's too confusing. Why not just move the implementation of this method to `setLogLevel(Map, LogAppenderInitializer)`?